@navikt/core/react/src/provider/theme/AkselTheme.tsx

```diff
@@ -0,0 +1,55 @@
+import cl from "clsx";
+import React, { forwardRef } from "react";
+import { createContext } from "../../util/create-context";
+
+type AkselThemeContext = {
+  appearance: "light" | "dark";
```

Comment:
Why not call it theme?
```suggestion
  theme: "light" | "dark";
```

Reply:
Im a little conflicted by what's the "most" correct naming for these cases:
- We have light and dark-mode
- We add support for High contrast (or some other)

What is the "theme" now? I could argue that we are using a "high-contrast"-theme with a dark "appearance". But might just be overcomplicating it. For now i changed it to "theme".

Reply:
Let's cross that bridge when we get to it 😄